demo/src/demos/ChartAdvisor/Log/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+
+import { Alert } from 'antd';
+
+import { ChartAdvisor } from '../../../../../packages/chart-advisor';
```

Comment:
oh, we can define alise in demo project later

Reply:
let's save it for an independent PR